webpack.config.babel.js

```diff
@@ -0,0 +1,48 @@
+import webpack from 'webpack';
+import path from 'path';
+
+const { NODE_ENV } = process.env;
+const production = NODE_ENV === 'production';
+
+const filename = `redux-actions${production ? '.min' : ''}.js`;
+
+const plugins = [
+  new webpack.optimize.OccurenceOrderPlugin(),
+  new webpack.DefinePlugin({
+    'process.env.NODE_ENV': JSON.stringify(NODE_ENV)
+  })
+];
+
+if (production) {
+  plugins.push(
+    new webpack.optimize.UglifyJsPlugin({
+      compressor: {
+        pure_getters: true,
+        unsafe: true,
+        unsafe_comps: true,
+        screw_ie8: true,
+        warnings: false
+      }
+    })
+  );
+}
+
+export default {
+  entry: path.join(__dirname, 'src/index.js'),
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename,
```

Comment:
What do you think about inlining `filename`? It's not used elsewhere, and when I got down here I had to look for its definition at the beginning of the file.